releasenotes/notes/0.25/display-control-flow-mpl-drawer-2dbc7b57ac52d138.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-features:
+features_visualization:
   - |
     In :class:`~qiskit.visualization.circuit.MatplotlibDrawer`, operations
```

Comment:
Normally for the drawer docstrings I point to `QuantumCircuit.draw()` and `qiskit.visualization.circuit_drawer()` and mention using `output='mpl'` for this kind of documentation. the `MatplotlibDrawer` class isn't really a user facing class that people directly interact with.